packages/database/src/migrations/20210222124639-UpdateLaosEntitiesPushMetadata-modifies-data.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NEW_METADATA = {
+  dhis: {
+    push: false,
+    isDataRegional: true,
+  },
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- We want to drop the entity trigger because we just want to add push: false to Laos entities, no actual data is changed, this shouldn't trigger any change listener (eg: for pushing)
+    DROP TRIGGER IF EXISTS entity_trigger
+    ON entity;
```

Comment:
We just want to add the `push` flag to metadata column, no actual data is changed. So dropping the trigger for entity within this transaction to prevent having 20000 entity change listeners fired